linux_pipeline/Jenkinsfile_container_images

```diff
@@ -1,28 +1,224 @@
 #!/usr/bin/env groovy
 
 def PowerShellWrapper(psCmd) {
+    psCmd = psCmd.replaceAll("\r", "").replaceAll("\n", "")
     bat "powershell.exe -NonInteractive -ExecutionPolicy Bypass -Command \"\$ErrorActionPreference='Stop';[Console]::OutputEncoding=[System.Text.Encoding]::UTF8;$psCmd;EXIT \$global:LastExitCode\""
 }
 
 pipeline {
   parameters {
-    string(defaultValue: "stable", description: 'Branch to be built', name: 'KERNEL_GIT_BRANCH')
+    choice(choices: "True\nFalse", description: 'Env cleanup', name: 'CLEAN_ENV')
+    choice(choices: "False\nTrue", description: 'Skip building stages', name: 'SKIP_BUILDING')
```

Comment:
Replace with ENABLED_STAGES and add the corresponding stages like in the other pipelines